### ISSUE TITLE
Add function to retrieve the remaining time in the current epoch interval

### ISSUE THREAD

**Issue by nathan-fiscaletti** [bellstrand/totp-generator#74]
I'd love to be able to generate a TOTP token with a key, but also know how much time is remaining in the current epoch interval (before that key will no longer be valid)

**Comment by nathan-fiscaletti** [comment]
This would (i think) be as simple as:
```js
const expires = (Math.ceil(epoch / period) * period);
```

**Comment by akwasin** [comment]
@nathan-fiscaletti did you figure out how to extract time left? Im looking at https://github.com/bellstrand/totp-generator/blob/master/index.js but cant figure it out

**Comment by C-Doug-iS** [comment]
```js
const seconds_until_expiration = period - (epoch % period)
```

I believe that would give you what you're looking for.

**Comment by bellstrand** [comment]
I've done a major rewrite since this required making a breaking change in the return type, and i've though about converting the lib to TypeScript for a while now.

I'll release it soon as long as I haven't messed anything up with the esm/cjs exports.

Untill then you can install using `npm install totp-generator@beta`.

**Comment by bellstrand** [comment]
Closing this PR as it's included in the `1.0.0` release.
